src/_P039_Thermosensors.ino

```diff
@@ -70,8 +71,8 @@
 # define PLUGIN_VALUENAME1_039 "Temperature"
 
 // typically 500ns of wating on positive/negative edge of CS should be enough ( -> datasheet); to make sure we cover a lot of devices we spend 1ms
-// FIX 2021-05-05: review of all covered device datasheets showed 1ms is more than enough; review with every newly added device
-#define P039_CS_Delay()             delayMicroseconds(1u)
+// FIX 2021-05-05: review of all covered device datasheets showed 2ms is more than enough; review with every newly added device
+#define P039_CS_Delay()             delayMicroseconds(2u)
```

Comment:
Comment suggests milli seconds, code suggests microseconds. Which one is it?

Reply:
Fair remark - solved in next push !